### PR TITLE
Fix section author markup

### PR DESCRIPTION
**Pull request by jaapio**
Apply the correct template for section authors.
Add an extra fix in the migration from old doc styles to correct the display of author names.

See: https://staging.doctrine-project.org/projects/doctrine-orm/en/3.3/cookbook/implementing-arrayaccess-for-domain-objects.html#implementing-arrayaccess-for-domain-objects

### CODE REVIEW

File: lib/Docs/RST/RSTCopier.php [hunk 1]
```diff
@@ -112,6 +112,14 @@ private function fixRSTSyntax(Project $project, string $content): string
         // get rid of .. include:: toc.rst
         $content = str_replace('.. include:: toc.rst', '', $content);
 
+        $content = preg_replace(
+            '/.. sectionauthor:: ([^\(]+) \(([^\)]+)\)/',
+            '.. sectionauthor:: $1 <$2>',
+            $content,
+        );
```
Comment (greg0ire) [inline lib/Docs/RST/RSTCopier.php:119]: Why is this necessary? Are there docs that don't follow the desired syntax already?

**Review by greg0ire — APPROVED** [review]

File: lib/Docs/RST/RSTCopier.php (hunk 1 above)
Comment (jaapio) [reply lib/Docs/RST/RSTCopier.php:119]: Yes, some of the old docs to use `jaapio (jaap@example.com)` where it should be `jaapio <jaap@example.com>`

**Review by SenseException — APPROVED** [review]
